keyboards/alpha/keymaps/hvp/keymap.c

```diff
@@ -0,0 +1,49 @@
+#include QMK_KEYBOARD_H
+
+#define HOME 0
```

Comment:
You may want to use an enum here, as it may be easier to maintain long term.

Reply:
```c
enum layer_names {
    HOME,
    MODS,
    MODS2,
    OTHER,
};